pcsx2/Counters.cpp

```diff
@@ -477,9 +477,9 @@ static __fi void DoFMVSwitch()
 	if (EmuConfig.Gamefixes.SoftwareRendererFMVHack && (GSConfig.UseHardwareRenderer() || (RendererSwitched && GSConfig.Renderer == GSRendererType::SW)))
 	{
 		RendererSwitched = GSConfig.UseHardwareRenderer();
-
+		DevCon.Warning("FMV Switch");
```

Comment:
Note to self, remove this before merging

Reply:
Tbh it could be useful to have, maybe in DevCon.